packaging/torchvision/meta.yaml

```
@@ -25,9 +24,8 @@ requirements:
     - python
     - libpng
     - ffmpeg >=4.2  # [not win]
-    - jpeg <=9b
+    - jpeg
     - pillow >=4.1.1
```

I forgot to update this in https://github.com/pytorch/vision/pull/3641 so we might as well do it now:

```suggestion
    - jpeg
    - pillow >=5.3.0
```